src/app/containers/MediaPlayer/index.jsx

```diff
@@ -173,35 +179,42 @@ const MediaPlayerContainer = ({
       <Caption block={captionBlock} type={mediaInfo.type} service={service} />
     ) : null;
 
+  const mediaPlayerType = () =>
```

Comment:
this doesn't need to be a function

Reply:
it can just be a variable

Reply:
I am very dumb 😂. Made the changes 😄. Thank you for the review 👍

Reply:
😆 you are not dumb! Also, I just realised the `renderCaption` example is doing the same. It can just be a var too called `caption`

Reply:
yep, fixed this too. Luckily that was not on me, feel tiny bit better 😂